src/main/java/fi/aalto/cs/apluscourses/ui/courseproject/CourseProjectView.form

```diff
@@ -1,82 +1,105 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="fi.aalto.cs.apluscourses.ui.courseproject.CourseProjectView">
-  <grid id="27dc6" binding="basePanel" layout-manager="GridLayoutManager" row-count="11" column-count="1" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+  <grid id="27dc6" binding="basePanel" layout-manager="GridLayoutManager" row-count="13" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```

Comment:
I tried to keep the layout as it was but that editor is difficult to use. Apparently there are some remnants left behind of my editing. Do you think it should be cleaned up?

Reply:
Agreed, it's a pain in the --- to use. It seems like the vertical space between the restart checkbox and the settings text was lost. It would be nice to have it there so the spacing between the items looks even, but it's not a must if it's too difficult to add

*Before:*
![dialog1](https://user-images.githubusercontent.com/30376763/84150051-01b33b00-aa6a-11ea-9ea9-67b94f5d5e17.png)
*After:*
![dialog2](https://user-images.githubusercontent.com/30376763/84150061-0546c200-aa6a-11ea-86df-9334ca47c6e3.png)

Reply:
Now that I look at it closer, somehow also a lot of empty space has appeared above the settings text in the new layout :thinking:

Reply:
You're right. My bad. I'll fix.

Reply:
![Annotation 2020-06-09 173540](https://user-images.githubusercontent.com/6066364/84161274-c61f6d80-aa77-11ea-8b2d-d2e29d6cc850.png)

I guess I fixed it.

Reply:
Looks good :heavy_check_mark: